etc/containers/systemd/ContainerFile.rear-camera

```diff
@@ -0,0 +1,15 @@
+# ContainerFile used to create the image available at quay.io/qm-images/multimedia:latest
```

Comment:
Ho did you build this multimedia latest?

Reply:
see line 5: podman build -t quay.io/qm-images/multimedia:latest -f ContainerFile (in this case ContainerFile.rear-camera)